src/wp-admin/includes/class-wp-media-list-table.php

```diff
@@ -842,9 +842,17 @@ private function _get_row_actions( $post, $att_title ) {
 					esc_url( wp_get_attachment_url( $post->ID ) ),
 					/* translators: %s: Attachment title. */
 					esc_attr( sprintf( __( 'Copy &#8220;%s&#8221; URL to clipboard' ), $att_title ) ),
```

Comment:
Does this line need to change for the `aria-label`?

Reply:
To a shorter one? I think the change on line `845` is made due to visual clutter, the `aria-label` is not visual, so I guess no change is needed.